src/Bpi/ApiBundle/Controller/ChannelController.php

```diff
@@ -36,22 +38,84 @@ class ChannelController extends BPIController
      */
     public function listChannelsAction()
     {
-        $channelRepository = $this->getRepository('BpiApiBundle:Entity\Channel');
+        $query = new \Bpi\ApiBundle\Domain\Entity\ChannelQuery();
+        $query->amount(20);
+        if (false !== ($amount = $this->getRequest()->query->get('amount', false))) {
```

Comment:
I'll change this.

However, the code is similar to the code in RestController.php where $this->getRequest()->query->get(…, false) is used. Maybe we should clean that up as well.